java/ql/src/semmle/code/java/security/UrlRedirect.qll

```diff
@@ -1,12 +1,14 @@
-import java
-import semmle.code.java.frameworks.Servlets
+/** Provides classes to reason about URL redirect attacks. */
+
 import semmle.code.java.dataflow.DataFlow
```

Comment:
Missing `import java`.

Reply:
Just for my understanding. Do we also want to have `import java` even though it is provided by for example `import semmle.code.java.dataflow.DataFlow`?

Reply:
Yes. Although this is perhaps more a matter of style since the type discovery order will likely be the same.  But better play it safe.